generator/testdata/google/example/library/v1/library_service.gapic.cc.baseline

```diff
@@ -3,90 +3,97 @@
+google::gax::StatusOr<::google::example::library::v1::ListBooksResponse>
 LibraryService::ListBooks(
 ::google::example::library::v1::ListBooksRequest const& request) {
   // TODO: actual useful work, e.g. retry, backoff, metadata, pagination, etc.
-  grpc::ClientContext context;
+  google::gax::CallContext context(ListBooks_info_);
   ::google::example::library::v1::ListBooksResponse response;
-  grpc::Status status = stub_->ListBooks(&context, request, &response);
-  if (status.ok()) {
+  google::gax::Status status = stub_->ListBooks(context, request, &response);
+  if (status.IsOk()) {
     return response;
   } else {
-    return gax::GrpcStatusToGaxStatus(status);
+    return status;
   }
 }
 
-gax::StatusOr<::google::example::library::v1::Empty>
+google::gax::StatusOr<::google::example::library::v1::Empty>
 LibraryService::DeleteBook(
 ::google::example::library::v1::DeleteBookRequest const& request) {
   // TODO: actual useful work, e.g. retry, backoff, metadata, pagination, etc.
-  grpc::ClientContext context;
+  google::gax::CallContext context(DeleteBook_info_);
   ::google::example::library::v1::Empty response;
-  grpc::Status status = stub_->DeleteBook(&context, request, &response);
-  if (status.ok()) {
+  google::gax::Status status = stub_->DeleteBook(context, request, &response);
+  if (status.IsOk()) {
     return response;
   } else {
-    return gax::GrpcStatusToGaxStatus(status);
+    return status;
   }
 }
 
-gax::StatusOr<::google::example::library::v1::Book>
+google::gax::StatusOr<::google::example::library::v1::Book>
 LibraryService::UpdateBook(
 ::google::example::library::v1::UpdateBookRequest const& request) {
   // TODO: actual useful work, e.g. retry, backoff, metadata, pagination, etc.
-  grpc::ClientContext context;
+  google::gax::CallContext context(UpdateBook_info_);
   ::google::example::library::v1::Book response;
-  grpc::Status status = stub_->UpdateBook(&context, request, &response);
-  if (status.ok()) {
+  google::gax::Status status = stub_->UpdateBook(context, request, &response);
+  if (status.IsOk()) {
     return response;
   } else {
-    return gax::GrpcStatusToGaxStatus(status);
+    return status;
   }
 }
 
-gax::StatusOr<::google::example::library::v1::Book>
+google::gax::StatusOr<::google::example::library::v1::Book>
 LibraryService::GetBigBook(
 ::google::example::library::v1::GetBookRequest const& request) {
   // TODO: actual useful work, e.g. retry, backoff, metadata, pagination, etc.
-  grpc::ClientContext context;
+  google::gax::CallContext context(GetBigBook_info_);
   ::google::example::library::v1::Book response;
-  grpc::Status status = stub_->GetBigBook(&context, request, &response);
-  if (status.ok()) {
+  google::gax::Status status = stub_->GetBigBook(context, request, &response);
+  if (status.IsOk()) {
     return response;
   } else {
-    return gax::GrpcStatusToGaxStatus(status);
+    return status;
   }
 }
 
+constexpr MethodInfo LibraryService::CreateBook_info_;
```

Comment:
Did you intent to have these be just declarations? Or should they be definitions?

Reply:
To the best of my understanding this is necessary to force the linker to allocate space for static constexpr objects.

I used the same pattern in the Operation PR after a lot of experimentation. I'm not sure I've fully internalized all the rules, but this finally got the linker to stop complaining to me.